spec/UtilsTest/streams-utils-spec.js

```diff
@@ -0,0 +1,35 @@
+'use babel';
+
+import * as path from 'path';
+import StreamsUtils from '../../lib/util/streams-utils';
+
+describe('streams-utils', () => {
+  describe('getFqnMainComposites', () => {
+    it('tests the getFqnMainComposites function\'s output', () => {
+      const fqnMainComposites = StreamsUtils.getFqnMainComposites(`${__dirname}${path.sep}..${path.sep}splFiles${path.sep}simple${path.sep}simpleApp.spl`);
+      const expectedOutput = { fqn: 'test', namespace: '', mainComposites: ['test'] };
+      expect(fqnMainComposites).toEqual(expectedOutput);
+    });
+  });
+  describe('getFqnMainComposites 2', () => {
+    it('tests the getFqnMainComposites (2) function\'s output', () => {
+      const fqnMainComposites = StreamsUtils.getFqnMainComposites(`${__dirname}${path.sep}..${path.sep}splFiles${path.sep}utils${path.sep}twoCompositesApp.spl`);
+      const expectedOutput = { fqn: '', namespace: '', mainComposites: ['testName1', 'testName2'] };
+      expect(fqnMainComposites).toEqual(expectedOutput);
+    });
+  });
+  describe('parseV4ServiceCredentials', () => {
```

Comment:
do not put real values here. the test does not need them, and credentials should not be in source control. You can put empty strings as the values for each field here.